foxpuppet/windows/browser/window.py

```diff
@@ -12,7 +12,8 @@
 
 class BrowserWindow(BaseWindow):
 
-    """Representation of a browser window."""
+    """Representation of a browser window.
```

Comment:
We are taking care of the ```BaseWindow``` creation behind the scenes so it shouldn't affect the end user.